docs/setup-guide/multi-node-dev-cluster.md

```diff
@@ -1,52 +1,73 @@
 # How to Setup a small Dev Cluster of multi-nodes
 
-It may be desired to setup a dev cluster having 2 or 3 nodes in order to play with comprehansive features of Arktos. The simple and easy-to-use arktos-up.sh feels short in this case; this doc describes the minimum effort to run a cluster havinbg 2 nodes (1 master, plus a worker who joins later).
+It may be desired to setup a dev cluster having more worker nodes in order to play with comprehensive features of Arktos. The simple and easy-to-use arktos-up.sh feels short in this case; this doc describes the minimum effort to run a cluster having 1 master node and extra worker nodes joining after the master is up.
 
-This doc, as at the moment written, does not mandate which cni plugin to be used. It is up to user, though we recommend to use calico, or our own alktron.
+This doc, as at the moment written, does not mandate which cni plugin to be used; it is up to the user. We have verified that Flannel works well with multi-tenancy Arktos cluster; the instructions laid out here is based on our experience with Flannel at GCP env.
 
-1. start the master node 
- the master node, run follwoing commands to start cluster, which has the master only.
+1. bootstrap the cluster by starting the master node (no CNI plgin at first)
 ```bash
+export ARKTOS_NO_CNI_PREINSTALLED=y
 ./hack/arktos-up.sh
 ```
 
-2. copy needed files to worker node
+2. install CNI plugin
+```bash
+./cluster/kubectl.sh apply -f https://github.com/coreos/flannel/raw/master/Documentation/kube-flannel.yml
+```
 
-following files are required to copy over from master to worker node:
-_output/local/bin/linux/amd64/hyperkube
-/var/run/kubernetes/kubelet.kubeconfig
-/var/run/kubernetes/kube-proxy.kubeconfig
-/var/run/kubernetes/client-ca.crt
-/tmp/kube-proxy.yaml
+3. on the GCP instance to be added as worker node, ensure follwoing worker secret files copied from the master node:
+```bash
+mkdir -p /tmp/arktos
+gcloud compute scp <master-node-instance>:/var/run/kubernetes/kubelet.kubeconfig /tmp/arktos/
+gcloud compute scp <master-node-instance>:/var/run/kubernetes/client-ca.crt /tmp/arktos/
+```
 
+At the moment this doc is written, kube-proxy does not support multi-tenancy yet, won't be deployed here. After kube-proxy issue has been fixed (we will allocate resource to cope with it soon), probably we will also need to copy over kube-proxy related secret and artifacts.
```

Comment:
The first sentence got a grammatical issue.

Reply:
will reword the relevant part.